### DISCUSSION TITLE
nothing I type inside firenvim frame is returned to reddit

### DISCUSSION THREAD

**Discussion by bogorad** [glacambre/firenvim#1602] [Q&A] [answered]
works fine with github, I'm typing this in firenvim, but in reddit everything disappears the moment I hit ZZ. what am i missing?

also, it there a way to make the text bigger?

**Comment by glacambre** [comment, marked as answer]
> but in reddit everything disappears the moment I hit ZZ

This is a [known issue](https://github.com/glacambre/firenvim/issues/672), reddit uses a fancy JavaScript editor that requires special handling, which I've been unable to implement. You need to switch to the simple editor, if that option still exists.

>also, it there a way to make the text bigger?

You can use [`:set guifont`](https://neovim.io/doc/user/options.html#'guifont').

**Reply by bogorad** [reply]
thank you!
